gradio/utils.py

```diff
@@ -1007,6 +1010,16 @@ def get_class_that_defined_method(meth: Callable):
         return cls.__name__
 
 
+def highlight_code(code, name, attrs):
+    try:
+        lexer = get_lexer_by_name(name)
+    except:
```

Comment:
🤔 This should have triggered a Ruff E722... But maybe the base for this PR was before #3979.